2012/10/18 03:07:19 [error] [php] Trying to get property of non-object (C:\xampp\htdocs\yii\yiiuser\protected\views\layouts\main.php:37)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\widgets\CContentDecorator.php(77): SiteController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\widgets\CContentDecorator.php(55): CContentDecorator->decorate()
#2 C:\xampp\htdocs\yii\framework\web\widgets\COutputProcessor.php(45): CContentDecorator->processOutput()
#3 C:\xampp\htdocs\yii\framework\web\CBaseController.php(207): CContentDecorator->run()
#4 C:\xampp\htdocs\yii\framework\web\CBaseController.php(301): SiteController->endWidget()
#5 C:\xampp\htdocs\yii\yiiuser\protected\views\layouts\column1.php(6): SiteController->endContent()
#6 C:\xampp\htdocs\yii\framework\web\CBaseController.php(127): require()
#7 C:\xampp\htdocs\yii\framework\web\CBaseController.php(96): SiteController->renderInternal()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(785): SiteController->renderFile()
#9 C:\xampp\htdocs\yii\yiiuser\protected\controllers\SiteController.php(32): SiteController->render()
#10 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(50): SiteController->actionIndex()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(309): CInlineAction->runWithParams()
#12 C:\xampp\htdocs\yii\framework\web\CController.php(287): SiteController->runAction()
#13 C:\xampp\htdocs\yii\framework\web\CController.php(266): SiteController->runActionWithFilters()
#14 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(283): SiteController->run()
#15 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(142): CWebApplication->runController()
#16 C:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#17 C:\xampp\htdocs\yii\yiiuser\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/yiiuser/
in C:\xampp\htdocs\yii\yiiuser\protected\views\layouts\main.php (37)
in C:\xampp\htdocs\yii\yiiuser\protected\views\layouts\column1.php (6)
in C:\xampp\htdocs\yii\yiiuser\protected\controllers\SiteController.php (32)
2012/10/18 03:08:03 [error] [php] Trying to get property of non-object (C:\xampp\htdocs\yii\yiiuser\protected\modules\user\models\User.php:36)
Stack trace:
#0 C:\xampp\htdocs\yii\yiiuser\protected\modules\user\models\User.php(28): model()
#1 C:\xampp\htdocs\yii\yiiuser\protected\modules\user\components\UserIdentity.php(27): model()
#2 C:\xampp\htdocs\yii\yiiuser\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#3 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(43): LoginForm->authenticate()
#4 C:\xampp\htdocs\yii\framework\validators\CValidator.php(214): CInlineValidator->validateAttribute()
#5 C:\xampp\htdocs\yii\framework\base\CModel.php(158): CInlineValidator->validate()
#6 C:\xampp\htdocs\yii\yiiuser\protected\controllers\SiteController.php(94): LoginForm->validate()
#7 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(50): SiteController->actionLogin()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(309): CInlineAction->runWithParams()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(287): SiteController->runAction()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(266): SiteController->runActionWithFilters()
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(283): SiteController->run()
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(142): CWebApplication->runController()
#13 C:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#14 C:\xampp\htdocs\yii\yiiuser\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/yiiuser/index.php/site/login
in C:\xampp\htdocs\yii\yiiuser\protected\modules\user\models\User.php (36)
in C:\xampp\htdocs\yii\yiiuser\protected\modules\user\models\User.php (28)
in C:\xampp\htdocs\yii\yiiuser\protected\modules\user\components\UserIdentity.php (27)
2012/10/18 03:08:24 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:287
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('gii')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\yiiuser\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/yiiuser/index.php/gii
---
2012/10/18 03:09:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:287
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('gii')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\yiiuser\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/yiiuser/index.php/gii
---
2012/10/18 03:10:14 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:287
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('gii')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\yiiuser\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/yiiuser/gii
---
2012/10/18 03:10:53 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:287
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('gii')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\yiiuser\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/yiiuser/gii
---
2012/10/18 03:10:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:287
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('gii')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\yiiuser\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/yiiuser/gii
---
2012/10/18 03:10:57 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:287
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('gii')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\yiiuser\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/yiiuser/gii
---
2012/10/18 03:10:59 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:287
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('gii')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\yiiuser\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/yiiuser/gii
---
2012/10/18 03:11:00 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:287
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('gii')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\yiiuser\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/yiiuser/gii
---
2012/10/18 03:14:37 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:287
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('gii')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\yiiuser\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/yiiuser/gii
---
2012/10/18 03:14:38 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:287
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('gii')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\yiiuser\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/yiiuser/gii
---
2012/10/18 03:14:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:287
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('gii')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\yiiuser\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/yiiuser/gii
---
2012/10/18 03:14:58 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:287
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('gii')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\yiiuser\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/yiiuser/gii
---
2012/10/18 03:14:59 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:287
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('gii')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\yiiuser\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/yiiuser/gii
---
2012/10/18 03:14:59 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:287
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('gii')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\yiiuser\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/yiiuser/gii
---
2012/10/18 03:15:00 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:287
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('gii')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\yiiuser\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/yiiuser/gii
---
2012/10/18 03:15:01 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:287
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('gii')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\yiiuser\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/yiiuser/gii
---
2012/10/18 03:15:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:287
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('gii')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\yiiuser\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/yiiuser/gii
---
2012/10/18 03:15:10 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:287
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('gii')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\yiiuser\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/yiiuser/index.php/gii
---
2012/10/18 03:42:59 [error] [exception.CException] exception 'CException' with message 'Application authorization manager must extend the RDbAuthManager class.' in C:\xampp\htdocs\yii\yiiuser\protected\modules\rights\components\RInstaller.php:57
Stack trace:
#0 C:\xampp\htdocs\yii\framework\base\CModule.php(388): RInstaller->init()
#1 C:\xampp\htdocs\yii\yiiuser\protected\modules\rights\RightsModule.php(200): CModule->getComponent('installer')
#2 C:\xampp\htdocs\yii\yiiuser\protected\modules\rights\controllers\InstallController.php(29): RightsModule->getInstaller()
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): InstallController->init()
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('rights')
#5 C:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#6 C:\xampp\htdocs\yii\yiiuser\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/yii/yiiuser/rights
---
2012/10/18 03:43:10 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "rights/install.php".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:287
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('rights/install....')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\yiiuser\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/yiiuser/rights/install.php
---
2012/10/18 03:43:15 [error] [exception.CException] exception 'CException' with message 'Application authorization manager must extend the RDbAuthManager class.' in C:\xampp\htdocs\yii\yiiuser\protected\modules\rights\components\RInstaller.php:57
Stack trace:
#0 C:\xampp\htdocs\yii\framework\base\CModule.php(388): RInstaller->init()
#1 C:\xampp\htdocs\yii\yiiuser\protected\modules\rights\RightsModule.php(200): CModule->getComponent('installer')
#2 C:\xampp\htdocs\yii\yiiuser\protected\modules\rights\controllers\InstallController.php(29): RightsModule->getInstaller()
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): InstallController->init()
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('rights/install')
#5 C:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#6 C:\xampp\htdocs\yii\yiiuser\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/yii/yiiuser/rights/install
---
2012/10/18 07:21:24 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "rights".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:287
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('rights')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\yiiuser\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/yiiuser/rights
---
2012/10/18 07:25:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "rights".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:287
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('rights')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\yiiuser\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/yiiuser/rights
---
2012/10/18 08:04:52 [error] [exception.CException] exception 'CException' with message 'CPhpAuthManager and its behaviors do not have a method or closure named "getAssignmentsByItemName".' in C:\xampp\htdocs\yii\framework\base\CComponent.php:266
Stack trace:
#0 C:\xampp\htdocs\yii\yiiuser\protected\modules\rights\components\RAuthorizer.php(285): CComponent->__call('getAssignmentsB...', Array)
#1 C:\xampp\htdocs\yii\yiiuser\protected\modules\rights\components\RAuthorizer.php(285): CPhpAuthManager->getAssignmentsByItemName('Admin')
#2 C:\xampp\htdocs\yii\yiiuser\protected\modules\rights\controllers\AssignmentController.php(51): RAuthorizer->getSuperusers()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(1145): AssignmentController->accessRules()
#4 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(283): CController->run('')
#9 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('rights')
#10 C:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\yii\yiiuser\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/yii/yiiuser/rights
---
2012/10/18 09:03:37 [error] [exception.CException] exception 'CException' with message 'Alias "message.MessageModule" is invalid. Make sure it points to an existing PHP file and the file is readable.' in C:\xampp\htdocs\yii\framework\YiiBase.php:318
Stack trace:
#0 C:\xampp\htdocs\yii\framework\YiiBase.php(198): YiiBase::import('message.Message...', true)
#1 C:\xampp\htdocs\yii\framework\base\CModule.php(283): YiiBase::createComponent('message.Message...', 'message', NULL, Array)
#2 C:\xampp\htdocs\yii\yiiuser\protected\views\layouts\main.php(36): CModule->getModule('message')
#3 C:\xampp\htdocs\yii\framework\web\CBaseController.php(127): require('C:\xampp\htdocs...')
#4 C:\xampp\htdocs\yii\framework\web\CBaseController.php(96): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii\framework\web\widgets\CContentDecorator.php(77): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#6 C:\xampp\htdocs\yii\framework\web\widgets\CContentDecorator.php(55): CContentDecorator->decorate('<div id="conten...')
#7 C:\xampp\htdocs\yii\framework\web\widgets\COutputProcessor.php(45): CContentDecorator->processOutput('<div id="conten...')
#8 C:\xampp\htdocs\yii\framework\web\CBaseController.php(207): COutputProcessor->run()
#9 C:\xampp\htdocs\yii\framework\web\CBaseController.php(301): CBaseController->endWidget('CContentDecorat...')
#10 C:\xampp\htdocs\yii\yiiuser\protected\views\layouts\column1.php(6): CBaseController->endContent()
#11 C:\xampp\htdocs\yii\framework\web\CBaseController.php(127): require('C:\xampp\htdocs...')
#12 C:\xampp\htdocs\yii\framework\web\CBaseController.php(96): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#13 C:\xampp\htdocs\yii\framework\web\CController.php(785): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#14 C:\xampp\htdocs\yii\yiiuser\protected\modules\user\controllers\ProfileController.php(20): CController->render('profile', Array)
#15 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(50): ProfileController->actionProfile()
#16 C:\xampp\htdocs\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#17 C:\xampp\htdocs\yii\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#18 C:\xampp\htdocs\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#19 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(283): CController->run('')
#20 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('user/profile')
#21 C:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#22 C:\xampp\htdocs\yii\yiiuser\index.php(13): CApplication->run()
#23 {main}
REQUEST_URI=/yii/yiiuser/index.php/user/profile
HTTP_REFERER=http://localhost/yii/yiiuser/index.php/user/login
---
2012/10/18 09:04:19 [error] [exception.CException] exception 'CException' with message 'Alias "message.MessageModule" is invalid. Make sure it points to an existing PHP file and the file is readable.' in C:\xampp\htdocs\yii\framework\YiiBase.php:318
Stack trace:
#0 C:\xampp\htdocs\yii\framework\YiiBase.php(198): YiiBase::import('message.Message...', true)
#1 C:\xampp\htdocs\yii\framework\base\CModule.php(283): YiiBase::createComponent('message.Message...', 'message', NULL, Array)
#2 C:\xampp\htdocs\yii\yiiuser\protected\views\layouts\main.php(36): CModule->getModule('message')
#3 C:\xampp\htdocs\yii\framework\web\CBaseController.php(127): require('C:\xampp\htdocs...')
#4 C:\xampp\htdocs\yii\framework\web\CBaseController.php(96): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii\framework\web\widgets\CContentDecorator.php(77): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#6 C:\xampp\htdocs\yii\framework\web\widgets\CContentDecorator.php(55): CContentDecorator->decorate('<div id="conten...')
#7 C:\xampp\htdocs\yii\framework\web\widgets\COutputProcessor.php(45): CContentDecorator->processOutput('<div id="conten...')
#8 C:\xampp\htdocs\yii\framework\web\CBaseController.php(207): COutputProcessor->run()
#9 C:\xampp\htdocs\yii\framework\web\CBaseController.php(301): CBaseController->endWidget('CContentDecorat...')
#10 C:\xampp\htdocs\yii\yiiuser\protected\views\layouts\column1.php(6): CBaseController->endContent()
#11 C:\xampp\htdocs\yii\framework\web\CBaseController.php(127): require('C:\xampp\htdocs...')
#12 C:\xampp\htdocs\yii\framework\web\CBaseController.php(96): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#13 C:\xampp\htdocs\yii\framework\web\CController.php(785): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#14 C:\xampp\htdocs\yii\yiiuser\protected\controllers\SiteController.php(32): CController->render('index')
#15 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(50): SiteController->actionIndex()
#16 C:\xampp\htdocs\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#17 C:\xampp\htdocs\yii\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#18 C:\xampp\htdocs\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#19 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(283): CController->run('')
#20 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('')
#21 C:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#22 C:\xampp\htdocs\yii\yiiuser\index.php(13): CApplication->run()
#23 {main}
REQUEST_URI=/yii/yiiuser/
---
2012/10/18 09:06:47 [error] [exception.CException] exception 'CException' with message 'Alias "message.MessageModule" is invalid. Make sure it points to an existing PHP file and the file is readable.' in C:\xampp\htdocs\yii\framework\YiiBase.php:318
Stack trace:
#0 C:\xampp\htdocs\yii\framework\YiiBase.php(198): YiiBase::import('message.Message...', true)
#1 C:\xampp\htdocs\yii\framework\base\CModule.php(283): YiiBase::createComponent('message.Message...', 'message', NULL, Array)
#2 C:\xampp\htdocs\yii\yiiuser\protected\views\layouts\main.php(36): CModule->getModule('message')
#3 C:\xampp\htdocs\yii\framework\web\CBaseController.php(127): require('C:\xampp\htdocs...')
#4 C:\xampp\htdocs\yii\framework\web\CBaseController.php(96): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii\framework\web\widgets\CContentDecorator.php(77): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#6 C:\xampp\htdocs\yii\framework\web\widgets\CContentDecorator.php(55): CContentDecorator->decorate('<div id="conten...')
#7 C:\xampp\htdocs\yii\framework\web\widgets\COutputProcessor.php(45): CContentDecorator->processOutput('<div id="conten...')
#8 C:\xampp\htdocs\yii\framework\web\CBaseController.php(207): COutputProcessor->run()
#9 C:\xampp\htdocs\yii\framework\web\CBaseController.php(301): CBaseController->endWidget('CContentDecorat...')
#10 C:\xampp\htdocs\yii\yiiuser\protected\views\layouts\column1.php(6): CBaseController->endContent()
#11 C:\xampp\htdocs\yii\framework\web\CBaseController.php(127): require('C:\xampp\htdocs...')
#12 C:\xampp\htdocs\yii\framework\web\CBaseController.php(96): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#13 C:\xampp\htdocs\yii\framework\web\CController.php(785): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#14 C:\xampp\htdocs\yii\yiiuser\protected\controllers\SiteController.php(32): CController->render('index')
#15 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(50): SiteController->actionIndex()
#16 C:\xampp\htdocs\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#17 C:\xampp\htdocs\yii\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#18 C:\xampp\htdocs\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#19 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(283): CController->run('')
#20 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('')
#21 C:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#22 C:\xampp\htdocs\yii\yiiuser\index.php(13): CApplication->run()
#23 {main}
REQUEST_URI=/yii/yiiuser/
---
2012/10/18 09:13:45 [error] [exception.CException] exception 'CException' with message 'Alias "message.MessageModule" is invalid. Make sure it points to an existing PHP file and the file is readable.' in C:\xampp\htdocs\yii\framework\YiiBase.php:318
Stack trace:
#0 C:\xampp\htdocs\yii\framework\YiiBase.php(198): YiiBase::import('message.Message...', true)
#1 C:\xampp\htdocs\yii\framework\base\CModule.php(283): YiiBase::createComponent('message.Message...', 'message', NULL, Array)
#2 C:\xampp\htdocs\yii\yiiuser\protected\views\layouts\main.php(36): CModule->getModule('message')
#3 C:\xampp\htdocs\yii\framework\web\CBaseController.php(127): require('C:\xampp\htdocs...')
#4 C:\xampp\htdocs\yii\framework\web\CBaseController.php(96): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii\framework\web\widgets\CContentDecorator.php(77): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#6 C:\xampp\htdocs\yii\framework\web\widgets\CContentDecorator.php(55): CContentDecorator->decorate('<div id="conten...')
#7 C:\xampp\htdocs\yii\framework\web\widgets\COutputProcessor.php(45): CContentDecorator->processOutput('<div id="conten...')
#8 C:\xampp\htdocs\yii\framework\web\CBaseController.php(207): COutputProcessor->run()
#9 C:\xampp\htdocs\yii\framework\web\CBaseController.php(301): CBaseController->endWidget('CContentDecorat...')
#10 C:\xampp\htdocs\yii\yiiuser\protected\views\layouts\column1.php(6): CBaseController->endContent()
#11 C:\xampp\htdocs\yii\framework\web\CBaseController.php(127): require('C:\xampp\htdocs...')
#12 C:\xampp\htdocs\yii\framework\web\CBaseController.php(96): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#13 C:\xampp\htdocs\yii\framework\web\CController.php(785): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#14 C:\xampp\htdocs\yii\yiiuser\protected\controllers\SiteController.php(32): CController->render('index')
#15 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(50): SiteController->actionIndex()
#16 C:\xampp\htdocs\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#17 C:\xampp\htdocs\yii\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#18 C:\xampp\htdocs\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#19 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(283): CController->run('')
#20 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('')
#21 C:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#22 C:\xampp\htdocs\yii\yiiuser\index.php(13): CApplication->run()
#23 {main}
REQUEST_URI=/yii/yiiuser/
---
2012/10/18 09:13:46 [error] [exception.CException] exception 'CException' with message 'Alias "message.MessageModule" is invalid. Make sure it points to an existing PHP file and the file is readable.' in C:\xampp\htdocs\yii\framework\YiiBase.php:318
Stack trace:
#0 C:\xampp\htdocs\yii\framework\YiiBase.php(198): YiiBase::import('message.Message...', true)
#1 C:\xampp\htdocs\yii\framework\base\CModule.php(283): YiiBase::createComponent('message.Message...', 'message', NULL, Array)
#2 C:\xampp\htdocs\yii\yiiuser\protected\views\layouts\main.php(36): CModule->getModule('message')
#3 C:\xampp\htdocs\yii\framework\web\CBaseController.php(127): require('C:\xampp\htdocs...')
#4 C:\xampp\htdocs\yii\framework\web\CBaseController.php(96): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii\framework\web\widgets\CContentDecorator.php(77): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#6 C:\xampp\htdocs\yii\framework\web\widgets\CContentDecorator.php(55): CContentDecorator->decorate('<div id="conten...')
#7 C:\xampp\htdocs\yii\framework\web\widgets\COutputProcessor.php(45): CContentDecorator->processOutput('<div id="conten...')
#8 C:\xampp\htdocs\yii\framework\web\CBaseController.php(207): COutputProcessor->run()
#9 C:\xampp\htdocs\yii\framework\web\CBaseController.php(301): CBaseController->endWidget('CContentDecorat...')
#10 C:\xampp\htdocs\yii\yiiuser\protected\views\layouts\column1.php(6): CBaseController->endContent()
#11 C:\xampp\htdocs\yii\framework\web\CBaseController.php(127): require('C:\xampp\htdocs...')
#12 C:\xampp\htdocs\yii\framework\web\CBaseController.php(96): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#13 C:\xampp\htdocs\yii\framework\web\CController.php(785): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#14 C:\xampp\htdocs\yii\yiiuser\protected\controllers\SiteController.php(32): CController->render('index')
#15 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(50): SiteController->actionIndex()
#16 C:\xampp\htdocs\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#17 C:\xampp\htdocs\yii\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#18 C:\xampp\htdocs\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#19 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(283): CController->run('')
#20 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('')
#21 C:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#22 C:\xampp\htdocs\yii\yiiuser\index.php(13): CApplication->run()
#23 {main}
REQUEST_URI=/yii/yiiuser/
---
2012/10/18 09:13:47 [error] [exception.CException] exception 'CException' with message 'Alias "message.MessageModule" is invalid. Make sure it points to an existing PHP file and the file is readable.' in C:\xampp\htdocs\yii\framework\YiiBase.php:318
Stack trace:
#0 C:\xampp\htdocs\yii\framework\YiiBase.php(198): YiiBase::import('message.Message...', true)
#1 C:\xampp\htdocs\yii\framework\base\CModule.php(283): YiiBase::createComponent('message.Message...', 'message', NULL, Array)
#2 C:\xampp\htdocs\yii\yiiuser\protected\views\layouts\main.php(36): CModule->getModule('message')
#3 C:\xampp\htdocs\yii\framework\web\CBaseController.php(127): require('C:\xampp\htdocs...')
#4 C:\xampp\htdocs\yii\framework\web\CBaseController.php(96): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii\framework\web\widgets\CContentDecorator.php(77): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#6 C:\xampp\htdocs\yii\framework\web\widgets\CContentDecorator.php(55): CContentDecorator->decorate('<div id="conten...')
#7 C:\xampp\htdocs\yii\framework\web\widgets\COutputProcessor.php(45): CContentDecorator->processOutput('<div id="conten...')
#8 C:\xampp\htdocs\yii\framework\web\CBaseController.php(207): COutputProcessor->run()
#9 C:\xampp\htdocs\yii\framework\web\CBaseController.php(301): CBaseController->endWidget('CContentDecorat...')
#10 C:\xampp\htdocs\yii\yiiuser\protected\views\layouts\column1.php(6): CBaseController->endContent()
#11 C:\xampp\htdocs\yii\framework\web\CBaseController.php(127): require('C:\xampp\htdocs...')
#12 C:\xampp\htdocs\yii\framework\web\CBaseController.php(96): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#13 C:\xampp\htdocs\yii\framework\web\CController.php(785): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#14 C:\xampp\htdocs\yii\yiiuser\protected\controllers\SiteController.php(32): CController->render('index')
#15 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(50): SiteController->actionIndex()
#16 C:\xampp\htdocs\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#17 C:\xampp\htdocs\yii\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#18 C:\xampp\htdocs\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#19 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(283): CController->run('')
#20 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('')
#21 C:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#22 C:\xampp\htdocs\yii\yiiuser\index.php(13): CApplication->run()
#23 {main}
REQUEST_URI=/yii/yiiuser/
---
2012/10/18 09:14:22 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "messages".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:287
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('messages')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\yiiuser\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/yiiuser/messages
---
2012/10/18 09:14:27 [error] [exception.CException] exception 'CException' with message 'Alias "message.MessageModule" is invalid. Make sure it points to an existing PHP file and the file is readable.' in C:\xampp\htdocs\yii\framework\YiiBase.php:318
Stack trace:
#0 C:\xampp\htdocs\yii\framework\YiiBase.php(198): YiiBase::import('message.Message...', true)
#1 C:\xampp\htdocs\yii\framework\base\CModule.php(283): YiiBase::createComponent('message.Message...', 'message', NULL, Array)
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(338): CModule->getModule('message')
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(277): CWebApplication->createController('message')
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('message')
#5 C:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#6 C:\xampp\htdocs\yii\yiiuser\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/yii/yiiuser/message
---
2012/10/18 09:16:49 [error] [exception.CException] exception 'CException' with message 'Alias "message.MessageModule" is invalid. Make sure it points to an existing PHP file and the file is readable.' in C:\xampp\htdocs\yii\framework\YiiBase.php:318
Stack trace:
#0 C:\xampp\htdocs\yii\framework\YiiBase.php(198): YiiBase::import('message.Message...', true)
#1 C:\xampp\htdocs\yii\framework\base\CModule.php(283): YiiBase::createComponent('message.Message...', 'message', NULL, Array)
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(338): CModule->getModule('message')
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(277): CWebApplication->createController('message')
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('message')
#5 C:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#6 C:\xampp\htdocs\yii\yiiuser\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/yii/yiiuser/message
---
2012/10/18 09:23:00 [error] [exception.CException] exception 'CException' with message 'Alias "message.MessageModule" is invalid. Make sure it points to an existing PHP file and the file is readable.' in C:\xampp\htdocs\yii\framework\YiiBase.php:318
Stack trace:
#0 C:\xampp\htdocs\yii\framework\YiiBase.php(198): YiiBase::import('message.Message...', true)
#1 C:\xampp\htdocs\yii\framework\base\CModule.php(283): YiiBase::createComponent('message.Message...', 'message', NULL, Array)
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(338): CModule->getModule('message')
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(277): CWebApplication->createController('message')
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('message')
#5 C:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#6 C:\xampp\htdocs\yii\yiiuser\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/yii/yiiuser/message
---
2012/10/18 09:23:01 [error] [exception.CException] exception 'CException' with message 'Alias "message.MessageModule" is invalid. Make sure it points to an existing PHP file and the file is readable.' in C:\xampp\htdocs\yii\framework\YiiBase.php:318
Stack trace:
#0 C:\xampp\htdocs\yii\framework\YiiBase.php(198): YiiBase::import('message.Message...', true)
#1 C:\xampp\htdocs\yii\framework\base\CModule.php(283): YiiBase::createComponent('message.Message...', 'message', NULL, Array)
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(338): CModule->getModule('message')
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(277): CWebApplication->createController('message')
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('message')
#5 C:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#6 C:\xampp\htdocs\yii\yiiuser\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/yii/yiiuser/message
---
2012/10/18 09:24:59 [error] [exception.CException] exception 'CException' with message 'Alias "message.MessageModule" is invalid. Make sure it points to an existing PHP file and the file is readable.' in C:\xampp\htdocs\yii\framework\YiiBase.php:318
Stack trace:
#0 C:\xampp\htdocs\yii\framework\YiiBase.php(198): YiiBase::import('message.Message...', true)
#1 C:\xampp\htdocs\yii\framework\base\CModule.php(283): YiiBase::createComponent('message.Message...', 'message', NULL, Array)
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(338): CModule->getModule('message')
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(277): CWebApplication->createController('message')
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('message')
#5 C:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#6 C:\xampp\htdocs\yii\yiiuser\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/yii/yiiuser/message
---
